src/spring_log.js

```diff
@@ -21,7 +21,7 @@ log.transports.console = (msg) => {
 	}
 };
 
-const logPath = `${springPlatform.writePath}/spring-launcher.log`;
+const logPath = `${springPlatform.writePath}/infolog.txt`; // Who needs spring-launcher.log anyway?
```

Comment:
won't be accepting this change

Reply:
When Spring crashes, launcher logs don't contain the stacktrace, so we have to ask users to find and upload the infolog.txt manually. So from this perspective, launcher logs are much less useful - that was the intent behind this change.

Reply:
Works for me: https://logs.springrts.com/logfiles/6974/ , what's the actual problem? 
Do they restart/close the launcher before trying to upload or what?

Reply:
No, it's something else. Here's a recent example, you can compare the launcher log and infolog: https://discord.com/channels/549281623154229250/549282926596980738/915083659965517844
(sorry, don't want to attach someone's logs in here)

Reply:
The link you sent has two logs, in which spring-launcher's log contains everything that infolog has and then a bit more. It's the infolog.txt that gets cut short, although neither of the two have a stacktrace. 

![image](https://user-images.githubusercontent.com/64901/145403268-b4ba404e-ba5c-4bef-a548-3f4ba9e5ff72.png)